src/helpers.js

```diff
@@ -57,7 +57,12 @@ exports.getTopic = (attributes, slots) => {
     }
 };
 
-exports.getPodcastDirective = (podcastUrl) => {
+exports.getStage = (functionName) => {
```

Comment:
We should be trying to make any amendments and improvements now while we are developing and not later as later may never come and it may never be re-written.